src/instructlab/sdg/llmblock.py

```diff
@@ -45,8 +46,13 @@ def __init__(
             "model": self.model,
             "temperature": 0,
             "max_tokens": 12000,
+            #"seed": 12345,  TBD
         }
 
+        # Whether the LLM server supports a list of input prompts
+        # and supports the n parameter to generate n outputs per input
+        self.server_supports_batched = server_supports_batched(client, model_id)
```

Comment:
The `FlowParams` in #64 would give us a place to do this once rather than for every `LLMBlock`, but that can be fixed up later

Reply:
See `PipelineContext` in #86 now